ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.scss

```
@@ -205,8 +205,8 @@
       height: 48px;
 
       &--default {
-        background-color: #777a87;
-        color: white;
+        background-color: var(--color-background-alternative);
+        color: var(--color-text-muted); //@TODO: Not sure
```

Thoughts @georgewrmarshall ?

---

I think we are starting to see if an outlier exists and it doesn't fit into the system. Instead of trying to push the system to fit the component we can make the component adhere to the system. e.g the account and network menus. They were awkward to work with, with the dark colors inside light theme. Once we converted it to use light theme colors it was a dream to work with. Same goes for this instance. What does it look like if we just convert it to use the light. theme colors. Can we achieve the same UI goals(contrast, visibility, accessibility etc) if it's in light mode? Probably fine 😁 

```suggestion
        color: var(--color-text-alternative);
```